### ISSUE TITLE
GPU accelerated MMFF94

### ISSUE THREAD

**Issue by tarungog** [openbabel/openbabel#2174]
Sorry don't know where else to ask this. Is there a plugin or something that lets you GPU accelerate MMFF94 force field minimizations, energy calculations, etc?

**Comment by ghutchis** [comment]
No, not at present. Several people tried, but the current code would need a good rewrite.

There are some ideas on a "bridge" from Open Babel to OpenMM, which has support for custom forces. It would certainly take some coding work.

BTW, the general mailing list is openbabel-discuss @ lists.sourceforge.net and there's a developer list at openbabel-devel @ lists.sourceforge.net
